addons/hr_timesheet/views/hr_timesheet_portal_templates.xml

```diff
@@ -100,6 +100,47 @@
                                         </t>
                                     </th>
                                 </t>
+                                <t t-elif="groupby == 'parent_task'">
+                                    <th colspan="6">
+                                        <span t-if="timesheets[0].parent_task_id" t-field="timesheets[0].parent_task_id.name"/>
+                                        <span t-else="">No Parent Task</span>
+                                    </th>
+                                    <th colspan="1" class="text-end text-muted">
+                                        <t t-if="is_uom_day">
+                                            Total: <span t-esc="timesheets._convert_hours_to_days(hours_spent)" t-options='{"widget": "timesheet_uom"}'/>
+                                        </t>
+                                        <t t-else="">
+                                            Total: <span t-esc="hours_spent" t-options='{"widget": "float_time"}'/>
+                                        </t>
+                                    </th>
```

Comment:
Since this is repeated three time below, I am wondering whether we can do the t-if and only change the top part for each case (an example top part):
```xml
<th colspan="6">
    <span t-if="timesheets[0].parent_task_id" t-field="timesheets[0].parent_task_id.name"/>
    <span t-else="">No Parent Task</span>
</th>
```

Reply:
I originally meant to do that bu I guess I forgot, done now :+1: